File: test/All c/c++/Competetive coding/Google Kickstart 2022/starter_code (3)/starter_code.cc
#include <bits/stdc++.h>
using namespace std;

// Complete the MinimumDeliveryTime function
int MinimumDeliveryTime(int R, int C, vector <string> grid) {
  int delivery_time = 0;
  // TODO: Add logic to calculate the minimum overall delivery time you can
  // get by building at most one new delivery office.
  return delivery_time;
}

int main() {
  int T;
  // Get the number of test cases
  cin >> T;
  for(int t = 1; t <= T; t++) {
    // Get the dimensions of the grid
    int R, C;
    cin >> R >> C;
    // Get each row of the grid
    vector <string> grid;
    for(int i = 0; i < R; i++) {
      string row;
      cin >> row;
      grid.push_back(row);
    }
    cout << "Case #" << t << ": " << MinimumDeliveryTime(R, C, grid) << "\n";
  }
  return 0;
}

File: test/All c 1/c++/Competetive coding/Google Kickstart 2022/starter_code (3)/starter_code.java
import java.io.*;
import java.util.*;

public class Solution {
  private static int minimumDeliveryTime(int[][] deliveryOffices) {
    // TODO: solve for and return the minimum overall delivery time you can get
    // by building at most one new delivery office
    return 0;
  }

  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    int T = scanner.nextInt();

    for (int testCase = 0; testCase < T; ++testCase) {
      int R = scanner.nextInt();
      int C = scanner.nextInt();
      int[][] deliveryOffices = new int[R][C];

      for (int row = 0; row < R; ++row) {
        String line = scanner.next();
        for (int col = 0; col < C; ++col) {
          deliveryOffices[row][col] = line.charAt(col) - '0'; // converts char to int
        }
      }

      System.out.println("Case #" + testCase + ": " + minimumDeliveryTime(deliveryOffices));
    }
  }
}
